config/crds/gceproviderconfig_v1alpha1_gceclusterproviderspec.yaml

```diff
@@ -4,12 +4,12 @@ metadata:
   creationTimestamp: null
   labels:
     controller-tools.k8s.io: "1.0"
-  name: gceclusterproviderconfigs.gceproviderconfig.k8s.io
+  name: gceclusterproviderspecs.gceproviderconfig.k8s.io
 spec:
   group: gceproviderconfig.k8s.io
```

Comment:
I'm wondering if we should try to have something to do with cluster api in there, and try to be less specific about config.

e.g. cluster-api is `cluster.k8s.io` - should we be `gce.cluster.k8s.io` ?

Reply:
I'd be happy to discuss that (the name was just taken verbotem from what we used pre-CRDs, but not that it's more visible to users, it's worth revisiting). Can you open a separate issue though, since bikeshedding on the resource name here isn't going to be very useful.

Reply:
Opened https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/75